### PR TITLE
Toyeeb dev consolidation

### PR DESCRIPTION
**Pull request by tobstech**

### CODE REVIEW

**Review by talk2morris — CHANGES_REQUESTED** [review]
Hi Toyeeb, your branch has conflict.please resolve conflicts

File: omod/src/main/webapp/resources/reports/dataset-DataEntrantStatistics.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<org.openmrs.module.reporting.dataset.definition.SqlDataSetDefinition id="1" uuid="4abc7a9b-07b9-4ca5-a406-08708bf63c3b" retired="false">
```
Comment (talk2morris) [inline omod/src/main/webapp/resources/reports/dataset-DataEntrantStatistics.xml:1]: can u take this file out for now?

**Review by talk2morris — CHANGES_REQUESTED** [review]

File: omod/src/main/webapp/resources/reports/dataset-DataEntrantStatistics.xml (hunk 1 above)
Comment (tobstech) [reply omod/src/main/webapp/resources/reports/dataset-DataEntrantStatistics.xml:1]: Okay Sir

Comment (tobstech) [reply omod/src/main/webapp/resources/reports/dataset-DataEntrantStatistics.xml]: Done removed

**Review by talk2morris — APPROVED** [review]

**Pull request merged**